.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+known_mpltoolkits = mpl_toolkits
+known_numpy = numpy
+known_pyplot = matplotlib.pyplot
+known_firstparty = matplotlib
+sections = FUTURE,STDLIB,THIRDPARTY,NUMPY,PYPLOT,MPLTOOLKITS,FIRSTPARTY,LOCALFOLDER
```

Comment:
In a few files we import from both `matplotlib` and `mpl_toolkits` and because they are sections here they are separated with a space. Is that what we want, or do we want to consider mpl_toolkits a part of matplotlib and therefore on the same import section level? I'd vote for equivalence because we are in the same package, but I don't have a strong preference at all. I'd also probably suggest matplotlib before mpl_toolkits in the ordering (alphabetical and primary namespace).

Here is what it looks like currently, which may be a bit too spread out?

```python
import matplotlib.pyplot as plt

import mpl_toolkits.axes_grid1

import matplotlib
```

Reply:
Agree on matplotlib before mpl_toolkits. Guess that there are arguments for both separating row or not, no strong opinion there, but I would not object skipping the empty row if possible (so a soft preference).

Reply:
`matplotlib.pyplot` lives next to `numpy` - I've changed this a bit to add a `pydata` section in the config, to make the intent clear. If we were using e.g. pandas or scipy they would live there too under general pydata packages that people are likely to import together at the top.

Good shout on toolkits, I've moved toolkits after `matplotlib`, and removed the line break beetween them.

Reply:
This seems to have lead to that the pyplot-numpy section is sorted in alphabetical order, so pyplot now comes before numpy. Although the pydata makes sense, is that the wanted outcome?

Maybe it doesn't make a real difference, just that it earlier was the other way around...